model/task/task.go

```diff
@@ -855,9 +855,10 @@ func (t *Task) cacheExpectedDuration() error {
 }
 
 // Mark that the task has been dispatched onto a particular host. Sets the
-// running task field on the host and the host id field on the task.
-// Returns an error if any of the database updates fail.
-func (t *Task) MarkAsDispatched(hostId, distroId, agentRevision string, dispatchTime time.Time) error {
+// running task field on the host and the host id field on the task. If the host
```

Comment:
`MarkAsHostDispatched marks...`

Reply:
Fixed. I also tweaked the comment slightly since the doc comment seems to be outdated/incorrect.